exercises/exercise-list.md

```diff
@@ -0,0 +1,278 @@
+# Java For Beginners - Exercises
+
+* [Topic 2 Exercises](#exercise-list-topic-2)
+* [Topic 3 Exercises](#exercise-list-topic-3)
+
+# Exercise List Topic 2
+
+This exercise list is to practice the content presented in [lecture 2](../lessons/lesson2)
+
+## \[VABO] Variables and Basic Operators
+### Printing Varaibles
+In a `main`, define a `String`, `Integer`, and `Boolean` variable.
+1. `System.out.println` (print) each of them separately.
+2. Print them together in a sentence (which may or may not make sense).
+
+### \[MAOP] Math Operations
+Write a program that has 2 Integer variables `a` and `b`, print the result of each of applying addition, subtraction, division, multiplication, and modulus of `a` to `b`.
+
+e.g given a = 10 and b = 2, print:
+
+```java
+//print 12 (a+b)
```

Comment:
I find this confusing, are we asking them to print "print <answer>" or just the answer? I think if we're giving them an example, the example solution should be complete as well (no etc)

Reply:
Indeedy, next sweep will improve the wording of the exercises and expected outputs